src/main/java/jsr223/kubernetes/KubernetesScriptEngine.java

```diff
@@ -140,6 +140,16 @@ else if (k8sDeleteOnly) {
     private void writeKubernetesManifestFile(String k8s_manifest) {
         // Write k8s manifest to file
         try {
+
+            for (Map.Entry<String, String> variable : variables.entrySet()) {
+                if (k8s_manifest.contains("${variables_" + variable.getKey() + "}")) {
```

Comment:
I believe Java Code Convention does not recommend to use such_type_of_names.
But as you want I do not insist.